.azure-pipelines/packages.yml

```diff
@@ -145,10 +147,11 @@ jobs:
     vmImage: ubuntu-16.04
 
   steps:
-  - task: DotNetCoreInstaller@0
+  - task: UseDotNet@2
     displayName: install dotnet core sdk
     inputs:
       version: $(dotnetCoreSdkVersion)
+      includePreviewVersions: true
```

Comment:
See [previous comment](https://github.com/DataDog/dd-trace-dotnet/pull/520/files#r330122460).

Reply:
Resolved in ee4eb0ba384e8578b29e95cf00eed67e7db94f53